pkg/blockstorage/zone/zone.go

```diff
@@ -277,10 +210,11 @@ func isNodeReady(node *v1.Node) bool {
 	return false
 }
 
-func sanitizeAvailableZones(availableZones map[string]struct{}, validZoneNames []string) map[string]struct{} {
+// SanitizeAvailableZones validates and updates a map of zones against a list of valid zone names
+func SanitizeAvailableZones(availableZones map[string]struct{}, validZoneNames []string) map[string]struct{} {
```

Comment:
does this need to be exported?

Reply:
There were existing tests in other packages (awsebs, gcepd etc.) that were doing some zone validation. I exported this to be able to maintain those tests